src/Tasks/Resources/Strings.resx

```diff
@@ -2781,6 +2781,14 @@
     <value>MSB3893: Could not use a link to copy "{0}" to "{1}".</value>
     <comment>{StrBegin="MSB3893: "} LOCALIZATION: {0} and {1} are paths.</comment>
   </data>
+  <data name="Copy.IOException">
+    <value>MSB3094: "Got {0} copying {1} to {2} and HR is {3}"</value>
+    <comment>{StrBegin="MSB3094: "} LOCALIZATION: {0} is exception.ToString(), {1} and {2} are paths, {3} is a number")</comment>
+  </data>
+  <data name="Copy.RetryingOnAccessDenied">
+    <value>"MSB3095: Retrying on ERROR_ACCESS_DENIED because environment variable MSBUILDALWAYSRETRY = 1"</value>
+    <comment>{StrBegin="MSB3095: "} LOCALIZATION: Do NOT translate MSBUILDALWAYSRETRY")</comment>
+  </data>
```

Comment:
Codes seem to be taken already: https://github.com/search?q=repo%3Adotnet%2Fmsbuild+MSB3094+MSB3095&type=code

Reply:
Good catch. TLDR: Should have used glasses when coding at night. Mistaken 0 and 8, with my font they are similar:
![image](https://github.com/dotnet/msbuild/assets/25249058/a5ebca79-3daa-460a-a885-0b5016516cd9)
Fixed.